arrows/vtk/applets/register_applets.cxx

```diff
@@ -9,7 +9,9 @@
 #include <vital/plugin_loader/plugin_loader.h>
 #include <vital/applets/applet_registrar.h>
 
+#ifdef KWIVER_ARROWS_VTK_APPLETS_COLOR_MESH_H
```

Comment:
I'm not sure how this works because this macro is not defined until you include the `color_mesh.h`